### ISSUE TITLE
Could not run @clerk/remix on CloudFlare Workers. Cannot use 'import.meta' outside a module Syntax Error

### ISSUE THREAD

**Issue by aerojeyenth** [clerkinc/javascript#952]
- [x] Review the documentation: https://docs.clerk.dev/
- [x] Search for existing issues: https://github.com/clerkinc/javascript/issues
- [x] Go through package changelog files.
- [x] Provide the Frontend API key from your application dashboard.

## Package + Version

- [ ] `@clerk/clerk-js`
- [ ] `@clerk/clerk-react`
- [ ] `@clerk/nextjs`
- [x] `@clerk/remix`
- [ ] `@clerk/types`
- [ ] `@clerk/themes`
- [ ] `@clerk/localizations`
- [ ] `@clerk/clerk-expo`
- [ ] `@clerk/backend`
- [ ] `@clerk/clerk-sdk-node`
- [ ] `@clerk/shared`
- [ ] `@clerk/fastify`
- [ ] `gatsby-plugin-clerk`
- [ ] `build/tooling/chore`
- [ ] other:

### Version:

```
2.2.15
```

### Browser/OS

macOS Big Sur

## Description
Hi great people at Clerk. I am super impressed with what Clerk is doing. I want to use it in my next application. We are building a job board in Remix and hosting on Cloudflare Workers. After following the documents I made sure that Clerk/Remix supports all the edge env including CF workers but when I tried to build the app using Miniflare(Wrangler 2.12.3) locally by following the steps in the official documentation, I got the following error at the 2nd step of Wrapping the App with ClearkApp. 

_`SyntaxError: Cannot use 'import.meta' outside a module`_

![image](https://user-images.githubusercontent.com/6432625/226208284-eb998eb2-bda2-441e-87a7-29e63c1a7e31.png)

interestingly the failure occurs only after the Build succeeded not sure if this is the problem of Miniflare/Wrangler or Remix or Clerk/Remix packages. I kindly request you to help me debug this. Thanks a lot.

**### Steps to reproduce:**

Please clone the repo to local machine into a local folder using 

`git clone https://github.com/aerojeyenth/demo-for-cf-error`

run 

`npm install`

after npm install is compete 

run `npm run dev` 

`Actual Behaviour:`

 npm run command fails with the following error:
 
 _`SyntaxError: Cannot use 'import.meta' outside a module`_

`Expected Behaviour:`

npm run command should run without any errors and the remix application should run locally.

**Comment by dimkl** [comment]
Hi @aerojeyenth. Thank you for reporting this. I would be very nice to also provide feedback about `Cloudlfare Wranger/Miniflare`.
Sorry i haven't made any comment yet, but i haven't started the investigation yet.

About the label: the use of `remix-issue` is meant for some internal stats (eg determine which framework has the most GH issues) and shows that this issue is related to either our integration (our codebase) or with the remix framework itself (remix upgrade introduced breaking changes).

**Comment by dimkl** [comment]
@aerojeyenth there is a open issue for remix and Cloudflare Workers.
https://github.com/clerkinc/javascript/issues/127#issuecomment-1476523791

**Comment by dimkl** [comment]
This seems to be a duplicate of https://github.com/clerkinc/javascript/issues/127#issuecomment-1476523791
I will close it and update the old one. If you investigate and have any issues with Cloudlfare Wranger/Miniflare please open another ticket to track it separately!

**Comment by dimkl** [comment]
Closing this ticket as it depends on remix issue: https://github.com/remix-run/remix/issues/5644

**Comment by aerojeyenth** [comment]
@dimkl Thank you so much for helping me out on this! It was very quick. Big thanks!
